modules/release-notes/pages/index.adoc

```diff
@@ -2,82 +2,98 @@
 
-* Moved xref:tigergraph-server:data-loading:data-streaming-connector/index.adoc[data connector] setup from `gadmin` to GSQL. Streamlined the user experience for creating loading jobs for data stored in AWS, GCP, and Azure, directly integrating the data streaming connector with GSQL and reducing the steps needed to load data.
-** Introduced a new `STREAM` data loading type that simplifies the xref:tigergraph-server:data-loading:index.adoc[data loading process] from remote data storage.
-** Data loading with a `STREAM` connector now automatically adds new files from remote storage in near-real time.
-** Significantly improved CPU usage after connector finishes data loading.
-* Added a new set of xref:gsql-ref:ddl-and-loading:functions/token/index.adoc[token functions for string manipulation].
-* `to_string()` now supports all primitive and primitiveAccum data types.
-* Enhanced multi-edge capability by adding support for batch deletion of multiedges, both with GSQL commands and as a xref:tigergraph-server:API:built-in-endpoints.adoc#delete-an-edge-by-source-target-edge-type-and-discriminator[REST endpoint].
-* Queries can now be drafted and saved as xref:gsql-ref:querying:query-operations.adoc[draft queries] before they are able to be installed as functional queries.
-* Improved and standardized `SHOW PACKAGE` so it follows the same behavior as other `SHOW` catalog commands.
+==== Schema
+* Full support for xref:3.9@:gsql-ref:ddl-and-loading:defining-a-graph-schema.adoc#_discriminator[
+multiple edges of the same type] between vertices, to simplify application design for time series data and other complex large data sets.
 
+==== Querying and Query Management
+* `to_string()` now supports all xref:3.9@gsql-ref:ddl-and-loading:attribute-data-types.adoc#_primitive_types[primitive] and xref:3.9@gsql-ref:querying:accumulators.adoc#_vertex_attached_accumulators
+[scalar accumulator] data types.
+* Added support for batch deletion of multiple edges between vertices, both with GSQL `DELETE` and as a xref:3.9@tigergraph-server:API:built-in-endpoints.adoc#_delete_an_edge_by_source_target_edge_type_and_discriminator[REST endpoint].
+* Standardized `SHOW PACKAGE` for xref:3.9@:graph-ml:using-an-algorithm.adoc#_packaged_template_queries[packaged queries] to follow the same behavior as other `SHOW` catalog commands.
 
 === JDBC Spark Connector
 
-* Added support for loading job monitoring on xref:data-loading:spark-connection-via-jdbc-driver.adoc[JDBC Spark connector] loading jobs.
-* Added the ability to assign a specific job ID to a JDBC loading job.
-* Added error limiting parameters to JDBC loading jobs to cancel loading automatically when errors surpass a certain threshold.
+* [Alex] Added support for loading job monitoring on xref:3.9@data-loading:spark-connection-via-jdbc-driver.adoc[JDBC Spark connector] loading jobs.
```

Comment:
```suggestion
* Added support for loading job monitoring on xref:3.9@data-loading:spark-connection-via-jdbc-driver.adoc[JDBC Spark connector] loading jobs. This allows users to check loading progress as a job is running, letting them catch errors before a job is completed.
```

Reply:
@alexthomasTG What does a User need to do to monitor the progress?  The referenced page doesn't mention monitoring. Would it be better to link to a different page that does address monitoring?